src/main/java/com/demo/classroom/Controller/FileUploadController.java

```diff
@@ -0,0 +1,91 @@
+package com.demo.classroom.Controller;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import com.demo.classroom.Entity.Student;
+import com.demo.classroom.Entity.Teacher;
+import com.demo.classroom.Repository.StudentRepository;
+import com.demo.classroom.Repository.TeacherRepository;
+
+@RestController
+@RequestMapping("/api")
+public class FileUploadController {
+
+    @Value("${file.upload-dir}")
+    private String uploadDir;
+
+    @Autowired
+    private StudentRepository studentRepository;
+
+    @Autowired
+    private TeacherRepository teacherRepository;
+
```

Comment:
Why are we adding a controller here? Is this within the scope of this story?

Reply:
Yes. This is out of this scope. I am going to stash it.